packages/terminal/src/browser/terminal-copy-on-selection-handler.ts

```diff
@@ -36,18 +36,18 @@ export class TerminalCopyOnSelectionHander {
     }
 
     private async clipBoardCopyIsGranted(): Promise<boolean> {
-          // Unfortunately Firefox doesn't support permission check `clipboard-write`, so let try to copy anyway,
-          if (isFirefox) {
+        // Unfortunately Firefox doesn't support permission check `clipboard-write`, so let try to copy anyway,
+        if (isFirefox) {
             return true;
         }
         try {
             // tslint:disable-next-line:no-any
             const permissions = (navigator as any).permissions;
-            const { state } = await permissions.query({name: 'clipboard-write'});
+            const { state } = await permissions.query({ name: 'clipboard-write' });
             if (state === 'granted') {
                 return true;
             }
-        } catch (e) {}
+        } catch (e) { }
```

Comment:
[minor]: you could omit `e`.

Reply:
Thanks! I tried your suggestion and it looks like it complains:

<div align='left'>

<img width="364" alt="Screen Shot 2019-12-05 at 8 06 15 AM" src="https://user-images.githubusercontent.com/40359487/70237939-35697d00-1736-11ea-81ed-4e0367728ebb.png">


</div>

I'll keep it as is for the moment.

Reply:
No  worries, for the next time; the syntax is `try { } catch { } finally { }` or `try { } catch (e) { } finally { }`